src/reducers/reported.reducers.js

```diff
@@ -0,0 +1,18 @@
+import actionTypes from '../constants/reported.contants';
+
+const initialState = {
+  reportedArticle: [],
+};
+
+const reportedArticles = (state = initialState, action) => {
+  switch (action.type) {
+    case actionTypes.GET_REPORTED_ARTICLE_SUCCESS:
+      return { ...state, reportedArticle: action.reports };
+    case actionTypes.GET_REPORTED_ARTICLE_FAILURE:
+      return { ...state };
```

Comment:
You can also an error field to the state so we can know when an error like this occurs @sojida

Reply:
Something like

```
return {
   ...state,
   hasError: true,
   getReportedArticlesError: action.payload
```